definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
can you please use feature toggle here as well. I mean this file should be renamed to for example:
`definitions/divorce/json/CaseEventToFields/CaseEventToFields-sol-brexit-prod.json`

so that when we do release we know this is what should be replaced but whatever we have in 
`definitions/divorce/json/CaseEventToFields/CaseEventToFields-sol-brexit-NONprod.json`

it makes releasing MUCH easier :)

Reply:
NONprod is obviously just "nonprod". I wanted to make it clear to see :)